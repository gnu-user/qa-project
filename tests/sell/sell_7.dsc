TITLE
    Sell Title Length (Invalid Length)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the sell 
    transaction. Enter a title which is 26 characters (one more than max) and 
    verify that it is rejected, and an error occurs.
DEPENDENCIES
    sell_4
