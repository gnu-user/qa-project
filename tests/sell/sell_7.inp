login
administrator
sell
Title is too long!!!