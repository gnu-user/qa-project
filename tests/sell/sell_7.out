Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title: 
Invalid event title, event title cannot exceed 19 characters.